lldb/source/Target/TargetList.cpp

```diff
@@ -263,7 +271,8 @@ Status TargetList::CreateTargetInternal(
   // If we have a valid architecture, make sure the current platform is
   // compatible with that architecture
   if (!prefer_platform_arch && arch.IsValid()) {
-    if (!platform_sp->IsCompatibleArchitecture(arch, false, &platform_arch)) {
+    ArchSpec compatible_arch;
+    if (!platform_sp->IsCompatibleArchitecture(arch, false, &compatible_arch)) {
```

Comment:
Why not just pass nullptr here?  What is the "compatible_arch" for?

Reply:
`nullptr` is probably fine. I copied & pasted this from line 276 (old) / 285 (new).